src/components/Activity/ActivityItem.js

```diff
@@ -108,7 +109,7 @@ const ItemContent = React.memo(
       {transformAddresses(text, (part, isAddress, index) =>
         isAddress ? (
```

Comment:
@bpierre when could this not be an address? If it is not an address could it be rendered with the `IdentityBadge` compact mode too? (trying to see if we can get rid of the `else`).

Reply:
[`transformAddresses()` splits the text by addresses](https://github.com/aragon/aragon/blob/af851dba876246eaa2892e4d1c1c203ff2a456d1/src/web3-utils.js#L209), which means that “something 0x0000000000000000000000000000000000000000 something else” becomes:

```jsx
["something", "0x0000000000000000000000000000000000000000", "something else"]
```

The callback is then called on each of these parts, so that the content can be assembled back. So the `else` is just text :-)